web/wp-content/themes/lfevents/library/lfe-functions.php

```diff
@@ -770,7 +770,9 @@ function lfe_get_language_selector( $background_style, $menu_text_color ) {
 	$my_current_lang = apply_filters( 'wpml_translated_language_name', null, $my_current_lang, false );
 
 	echo '<li class="page_item page_item_has_children language-selector">';
-	echo '<a href="#">' . esc_html( $my_current_lang ) . '</a>';
+	echo '<a href="#">';
+	echo '<svg viewBox="0 0 24 24" width=32 ><path fill="currentColor" d="m11.62965 16.61452c-1.13922-.692-3.111-2.36313-3.153-2.32718a28.32942 28.32942 0 0 1 -3.30095 2.26177c-.68823.39708-1.38892.49615-1.82064-.09139a.992.992 0 0 1 .26656-1.40406c.00852-.00391 2.44665-1.594 3.25973-2.29678a11.64387 11.64387 0 0 1 -2.23281-3.53521 1.07774 1.07774 0 0 1 .52716-1.36835c.52715-.22205 1.049-.12664 1.48663.61989a10.33341 10.33341 0 0 0 1.8143 2.89517 10.853 10.853 0 0 0 2.1563-4.3469l-7.63293-.02148v-2.00685h4.8124v-.99406a.98574.98574 0 1 1 1.9713 0v.99406h5.1703v2.00685h-2.08646a17.03869 17.03869 0 0 1 -2.64065 5.75689 15.88157 15.88157 0 0 0 2.30149 1.66068l2.3092-5.66617a1.162 1.162 0 0 1 2.1802.01591l3.01041 7.389 1.85638 4.385h-2.47393l-1.08252-2.53924h-4.84082l-.888 2.53924h-2.5993l.287-.69166zm4.31307-5.16715-1.67531 4.55419h3.35059z"></path></svg>';
```

Comment:
This isn't valid in the SVG? - width=32

```
<svg viewBox="0 0 24 24" width="32"><path fill="currentColor" d="m11.62965 16.61452c-1.13922-.692-3.111-2.36313-3.153-2.32718a28.32942 28.32942 0 0 1 -3.30095 2.26177c-.68823.39708-1.38892.49615-1.82064-.09139a.992.992 0 0 1 .26656-1.40406c.00852-.00391 2.44665-1.594 3.25973-2.29678a11.64387 11.64387 0 0 1 -2.23281-3.53521 1.07774 1.07774 0 0 1 .52716-1.36835c.52715-.22205 1.049-.12664 1.48663.61989a10.33341 10.33341 0 0 0 1.8143 2.89517 10.853 10.853 0 0 0 2.1563-4.3469l-7.63293-.02148v-2.00685h4.8124v-.99406a.98574.98574 0 1 1 1.9713 0v.99406h5.1703v2.00685h-2.08646a17.03869 17.03869 0 0 1 -2.64065 5.75689 15.88157 15.88157 0 0 0 2.30149 1.66068l2.3092-5.66617a1.162 1.162 0 0 1 2.1802.01591l3.01041 7.389 1.85638 4.385h-2.47393l-1.08252-2.53924h-4.84082l-.888 2.53924h-2.5993l.287-.69166zm4.31307-5.16715-1.67531 4.55419h3.35059z"></path></svg>
```

Why is Japanese symbol suitable to mean "translations"?

Reply:
I'm not sure but it seems to be [the standard icon for "translations"](https://www.google.com/search?sca_esv=3532b07490db4deb&sxsrf=ACQVn0-3qHJ3rsvud3FdrwrpdLFmE3e9nA:1708415360683&q=translation+icon+svg&tbm=isch&source=lnms&sa=X&sqi=2&pjf=1&ved=2ahUKEwiK386Vt7mEAxVbUGwGHfwaBaAQ0pQJegQICxAB&biw=1440&bih=777&dpr=2).

Reply:
I didn't even realise the 2nd character was an A as it was joined. Joining the characters together seems to be the exception rather than the rule. We can do better. 

![Screenshot-2024-02-20 --14 52 06](https://github.com/LF-Engineering/lfevents/assets/10615884/c496f817-c6c1-47d1-96ea-b2b97dd48f7a)

Reply:
I kinda like it. I stole it from [jetbrains](https://www.jetbrains.com/).

Reply:
I could place the current language after the icon in the topnav, to make it more clear. I had it this way initially but figured it wasn't worth the space.
<img width=300 src="https://github.com/LF-Engineering/lfevents/assets/463160/b1f1af64-e47a-4195-a1c8-e42603fb4fb6">

Reply:
Personally I always find it a weird design pattern to inform the user what language is currently displayed so I don't see any benefit in it.